hr_expense_cancel/models/hr_expense.py

```diff
@@ -153,7 +153,7 @@ def _remove_move_reconcile(self, payments, account_move):
 
     def _cancel_payments(self, payments):
         for rec in payments:
-            rec.move_id.with_context({"force_delete": True}).unlink()
+            rec.move_id.button_cancel()
```

Comment:
Are these payments later reused?

Reply:
I honestly don't know (I'd have to check it out).

Before the change of odoo with this code the payment was deleted but now it is not deleted and I sincerely believe that it is the right thing to do and that what should be done is to cancel the move (which also cancel the payment).